docs/rst/replaying/usage/configuration.rst

```diff
@@ -131,6 +131,10 @@ For more information on topics, please read the `Fast DDS Topic <https://fast-dd
         - ``0`` (unlimited)
         - :ref:`replayer_max_tx_rate`
 
+.. warning::
+
+    The ``TRANSIENT_LOCAL`` durability is not compatible with the ``BEST_EFFORT`` reliability.
```

Comment:
Apart from this general Fast-DDS fact, which is not bad to have, I was referring more to the concrete case in which only one of them is set via specs/participant_qos/manual_topics while the other gets set through discovery.